components/aws/sagemaker/tests/integration_tests/resources/config/assume-role-training/config.yaml

```diff
@@ -0,0 +1,33 @@
+PipelineDefinition: resources/definition/training_pipeline.py
```

Comment:
thanks for the tests. We need more coverage since code change is going into multiple components

At least unit tests which tests main with assume role and good to have integ tests like these

Reply:
You want one more integration test as well?